william000000/AutoMart
# Bg fix all errors in testing 166813791

## What does this PR do?

- Fix all bugs about testing and set up Travis

## Description of the task to be completed

- A passing badge on my readme
- Make code cleaner(Remove consoles and repetition code) 
- Set up swagger documentation
- Update Readme
- Correct malfunction of endpoints
## What are relevant pivotal tracker stories
 [Fix bugs #166813791](https://www.pivotaltracker.com/story/show/166813791)

## How can it be manually tested
- clone my repo then after checkout to this branch `bg-fix-all-errors-in-testing-166813791` 
- Install all dependencies by this command `npm install` 
- create database `automart`
- export database `export DATABASE_URL`
- `npm run dev`
- After all the above steps, you can check any route and then check in a database for the result
